clang/lib/Sema/SemaTemplateInstantiateDecl.cpp

```diff
@@ -582,6 +582,17 @@ static void instantiateIntelSYCLFunctionAttr(
                                                   Result.getAs<Expr>());
 }
 
+static void instantiateSYCLIntelSchedulerTargetFmaxMhzAttr(
```

Comment:
Can we re-use `instantiateIntelSYCLFunctionAttr` defined above instead?

Reply:
Yes. Done.